mbedtls/tests/client_server.rs

```diff
@@ -16,13 +16,14 @@ use mbedtls::rng::CtrDrbg;
 use mbedtls::ssl::config::{Endpoint, Preset, Transport};
 use mbedtls::ssl::{Config, Context};
 use mbedtls::x509::{Certificate, LinkedCertificate, VerifyError};
+use mbedtls::Error;
 use mbedtls::Result as TlsResult;
 
 mod support;
 use support::entropy::entropy_new;
 use support::keys;
 
-fn client(mut conn: TcpStream) -> TlsResult<()> {
+fn client(mut conn: TcpStream, min_minor: i32, max_minor: i32, exp_minor: i32) -> TlsResult<()> {
```

Comment:
Please use `exp_minor: Result<i32, ()>` instead of -1 to indicate intent

Reply:
Done